mods/cnc/chrome/multiplayer-createserver.yaml

```diff
@@ -92,6 +92,15 @@ Container@MULTIPLAYER_CREATESERVER_PANEL:
 							Height: 20
 							Font: Regular
 							Text: Automatic port forwarding
+						Label@UPNP_NOTICE:
+							X: 130
+							Y: 180
+							Width: 200
+							Height: 25
+							Font: Tiny
+							Align: Left
+							WordWrap: True
+							Text: Requires Network Discovery (UPnP). Enable in Settings - Advanced.
```

Comment:
We disable UPnP (discovery) by default, because @chrisforbes and @obrakmann don't think it is polite to add holes to network security without asking first. You are confusing this with the `AllowPortForward` and `NatDeviceAvailable` settings.